src/pages/about.vue

```diff
@@ -0,0 +1,164 @@
+                >{{ $t('about.description.licenses-link') }}</a
+              >
+            </template>
+          </i18n>
+          <i18n path="about.collection" tag="p">
+            <template v-slot:common-crawl>
+              <a aria-label="common crawl" href="http://commoncrawl.org/"
+                >Common Crawl</a
+              >
+            </template>
+          </i18n>
+          <i18n path="about.planning.content" tag="p">
+            <template v-slot:vision>
+              <a
+                :aria-label="$t('about.aria.vision')"
+                href="https://creativecommons.org/2019/03/19/cc-search/"
+                >{{ $t('about.planning.vision') }}</a
+              >
+            </template>
+            <template v-slot:roadmap>
+              <a
+                :aria-label="$t('about.aria.roadmap')"
+                href="https://docs.google.com/document/d/19yH2V5K4nzWgEXaZhkzD1egzrRayyDdxlzxZOTCm_pc/edit#heading=h.jih78emira0r"
+                >{{ $t('about.planning.roadmap') }}</a
+              >
+            </template>
+            <template v-slot:working>
+              <a
+                :aria-label="$t('about.aria.projects')"
+                href="https://github.com/orgs/creativecommons/projects/7"
+                >{{ $t('about.planning.working') }}</a
+              >
+            </template>
+            <template v-slot:search>
+              <a
+                aria-label="cc search repository"
+                href="https://github.com/creativecommons/cccatalog-frontend/"
+                >{{ $t('about.planning.search') }}</a
+              >
+            </template>
+            <template v-slot:catalog-api>
+              <a
+                aria-label="cc catalog-api repository"
+                href="https://github.com/creativecommons/cccatalog-api/"
+                >{{ $t('about.planning.catalog-api') }}</a
+              >
+            </template>
+            <template v-slot:catalog>
+              <a
+                aria-label="cc catalog repository"
+                href="https://github.com/creativecommons/cccatalog/"
+                >{{ $t('about.planning.catalog') }}</a
+              >
+            </template>
+            <template v-slot:community>
+              <a
+                :aria-label="$t('about.aria.contribution')"
+                href="https://creativecommons.github.io/contributing-code/"
+                >{{ $t('about.planning.community') }}</a
+              >
+            </template>
+          </i18n>
+          <i18n path="about.declaration.content" tag="p">
+            <template v-slot:terms>
+              <a
+                :aria-label="$t('about.aria.terms')"
+                href="https://creativecommons.org/terms/"
+                >{{ $t('about.declaration.terms') }}</a
+              >
+            </template>
+          </i18n>
+          <i18n path="about.old-cc-search" tag="p">
+            <template v-slot:link>
```

Comment:
If wanted, you could use the shorthand for `v-slot:` which is `#`
```suggestion
            <template #link>
```